sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/VectorizedDeltaBinaryPackedReader.java

```diff
@@ -300,7 +300,8 @@ private void unpackMiniBlock() throws IOException {
         bitWidths[currentMiniBlock]);
     for (int j = 0; j < miniBlockSizeInValues; j += 8) {
```

Comment:
Testing the performance of the following changes:

```
  private void unpackMiniBlock() throws IOException {
    Arrays.fill(this.unpackedValuesBuffer, 0);
    BytePackerForLong packer = Packer.LITTLE_ENDIAN.newBytePackerForLong(
        bitWidths[currentMiniBlock]);
    int bitWidth = packer.getBitWidth();
    int times = miniBlockSizeInValues / 8;
    int total = times * bitWidth;
    ByteBuffer buffer = in.slice(total);
    byte[] array = buffer.array();
    int i = 0;
    int srcOffset = buffer.arrayOffset() + buffer.position();
    int targetOffSet = 0;
    while (i < times) {
      packer.unpack8Values(array, srcOffset, unpackedValuesBuffer, targetOffSet);
      i++;
      srcOffset += bitWidth;
      targetOffSet += 8;
    }
    remainingInMiniBlock = miniBlockSizeInValues;
    currentMiniBlock++;
  }
```

Reply:
after tests, there is no significant performance difference, keep the minimum change

Reply:
`SingleBufferInputStream#slice`  has fewer calls, but there is no significant performance difference